doc/install.text

```diff
@@ -0,0 +1,124 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+
+* Install lsst binary distribution (https://pipelines.lsst.io/install/conda.html)
+* Install git-lfs (https://git-lfs.github.com/)
+* Install matplotlib:
+
+conda install matplotlib
+
+
+If you have a very fresh compiler version, this could lead to incompatibilities 
+with the precompiled LSST stack. In case you run into trouble compiling or testing
+the LSST stack please install your compiler in anaconda:
```

Comment:
"Recent" rather than "fresh"?

Can we be more specific about exactly what versions of the compiler are required, and what the error is?

Alternatively: is there any downside to simply requiring that the user always install gcc through Conda? I don't think this can do any harm, and avoids a class of problems.